js/interface.js

```diff
@@ -2536,6 +2551,14 @@ $('[data-toggle="tooltip"]').tooltip({
   delay: { "show": 500, "hide": 300 }
 });
 
+$('[data-template="fl-store-releaseNotes"]').on('click', function() {
```

Comment:
Use the `a` element instead.

```suggestion
$('[data-template="fl-store-releaseNotes"] a').on('click', function(e) {
  e.preventDefault();
```

Alternatively, move the `data-template` attribute to the `a` element. By having click actions assigned to the `a` element, it helps with accessibility and navigation with tabs.